package.json

```diff
@@ -12,8 +12,10 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "classnames": "^2.5.1",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-responsive": "^10.0.0"
```

Comment:
I think we don't need to use any third-party libraries for such a simple functionality